docs/conditions.md

```diff
@@ -0,0 +1,73 @@
+# CR Conditions and Readiness Probe
+Conditions are..
+	   _the latest available observations of an object's state. They are
+	   an extension mechanism intended to be used when the details of an
+	   observation are not a priori known or would not apply to all
+	   instances of a given Kind._
+
+The HCO’s CR is a representation of the underlying component operators.  If the
+object exists, then all components exist.  If the object doesn’t exist, all
+components don’t exist.  However, the CR existence doesn’t help us with the
+state where the operators exist, but are they healthy?  This is where conditions
+on the HCO’s CR will answer this question by providing the observed health of
+the underlying components.
+
+## Condition List
+We can use some of the CVO's [conditions](https://github.com/openshift/api/blob/master/config/v1/types_cluster_operator.go#L121-L133) to standardize across components.
+
+#### OperatorAvailable
+```
+	OperatorAvailable ClusterStatusConditionType = "Available"
+```
+OperatorAvailable indicates that the binary maintained by the operator
+(eg: openshift-apiserver for the openshift-apiserver-operator), is functional
+and available in the cluster.
+
+#### OperatorProgressing
+```
+	OperatorProgressing ClusterStatusConditionType = "Progressing"
+```
+Progressing indicates that the operator is actively making changes to the binary
+maintained by the operator (eg: openshift-apiserver for the
+openshift-apiserver-operator).
+
+#### OperatorDegraded
+```
+	OperatorDegraded ClusterStatusConditionType = "Degraded"
+```
+Degraded indicates that the component operator is not functioning completely.
+An example of a degraded state would be if there should be 5 copies of the
+component running but only 4 are running. It may still be available, but it is
+degraded.
+
+#### Condition Matrix
```

Comment:
Maybe mention what is the default value of (missing) condition.

Reply:
What do you mean by 
> default value of (missing) condition
>
Are you asking what will happen if the HCO notices a CR hasn't implemented conditions or a particular condition?

Reply:
Every CR is initially without any state - conditions. If HCO sees this initial state, how does it evaluates condition value? True, False or Unknown.

Reply:
If the HCO sees the CR with no value for the conditions, then the HCO will assume the Operator's condition is `OperatorAvailable = false`, `OperatorProgressing = unknown` and `OperatorDegraded = unknown`.  This is a good place for `unknown`